mocks/logger/mock_logger.go

```diff
@@ -0,0 +1,238 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: utils/logger/logger.go
+
+// Package mocks is a generated GoMock package.
+package mock_logger
+
+import (
+	http "net/http"
+	reflect "reflect"
+
+	gomock "github.com/golang/mock/gomock"
+)
+
+// MockLoggerI is a mock of LoggerI interface
+type MockLoggerI struct {
```

Comment:
We usually name our structs with T in the end like MockLoggerT

Reply:
actually this is an auto-generated file that creates a struct according to the name of the interface